.travis.yml

```diff
@@ -5,14 +5,15 @@ services:
 
 language: go
 go:
-  - "1.10.x"
```

Comment:
This is fine, but it the badge in the README needs to be updated as well.

Reply:
Yep, I'll adapt.